src/hotp.rs

```diff
@@ -77,6 +86,10 @@ impl HOTP {
         false
     }
 
+    pub fn base32_secret(&self) -> String {
```

Comment:
Make this a `pub(crate)`?

Reply:
Actually this method is quite useful for users of the crate. You can create the secret using securely-generated random bytes, and then you can get the base32 secret which is the standard format (other than QR codes) for giving the secret to users.